test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -389,6 +392,48 @@ private async Task ExecuteAndValidate<T>(Func<Task<FhirResponse<T>>> action, str
                 ae => ValidateExecutedAuditEntry(ae, expectedAction, expectedResourceType, expectedUri, expectedStatusCode, correlationId, expectedAppId, ExpectedClaimKey));
         }
 
+        private async Task ExecuteAndValidateBatch<T>(Func<Task<FhirResponse<T>>> action, List<(string, string, HttpStatusCode?, ResourceType?)> expectedList)
```

Comment:
nit: use named tuples to increase readability (e.g. something like `(string auditAction, string route, HttpStatusCode? statusCode, ResourceType resourceType?)`)